Filters/exceptions.txt

```diff
@@ -1,3 +1,6 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1822
+@@||daraz.pk^|
+@@||daraz.com^|
```

Comment:
It will unblock for example `||marketing.admin.net.daraz.com^`. Maybe without subdomains?

```suggestion
@@|daraz.pk^|
@@|daraz.com^|
```

Reply:
Regarding `daraz.pk`, I cannot find a rule which block it, but if exception is required then I guess that we should also add:
```adblock
@@|www.daraz.pk^|
```
or not?

Reply:
> Regarding `daraz.pk`, I cannot find a rule which block it

I got it from the Private DNS log. Don't know why is that.
```adblock
||daraz.com^
```

Reply:
If `daraz.pk` is blocked by `||daraz.com^` then I think that it's weird (unless it's blocked by CNAME, but it's rather not).

Reply:
I think it's CNAME, same log as @zloyden

Reply:
> Regarding `daraz.pk`, I cannot find a rule which block it, but if exception is required then I guess that we should also add:
> 
> ```
> @@|www.daraz.pk^|
> ```
> 
> or not?

don't quite understand. Could you explain why that is worth adding?

Reply:
I mean that `@@|daraz.pk^|` doesn't unblock `www.daraz.pk` but only `daraz.pk` and on screenshot from the report (https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1822) there is `www.daraz.pk` blocked.
When I open `daraz.pk` then I'm also redirected to `www.daraz.pk`.

Reply:
@jellizaveta without unblocking `www.`, the site will not work.

![image](https://github.com/user-attachments/assets/77512d30-2476-40d6-8559-bd29c60fa634)

Please test with AdGuard Home or private.

Reply:
The site was loaded fine before, but I can confirm that the exception is needed.